sirepo/template/template_common.py

```diff
@@ -243,10 +244,17 @@ def __set_from_pc(cls, particle, energy):
 
 def analysis_job_dispatch(data):
     t = sirepo.template.import_module(data.simulationType)
+    d = simulation_db.simulation_run_dir(
```

Comment:
Passing data should work. parse_model looks for computeModel so you don't need this extra pkdict.